hs/Database/Database.hs

```diff
@@ -28,7 +29,8 @@ setupDatabase = do setupDistributionTable
                    print "breadth table set up"
                    parseAll
                    seedVideos
-
+                   getDepartments
+                   
```

Comment:
I've updated the file to get rid of the white space however it doesn't show this as an outdated diff unlike the rest of the issues below. Am I missing something? Thanks.